extensions/keycloak-admin-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -2,10 +2,9 @@
 name: "Keycloak Admin Client"
 metadata:
   keywords:
-  - "oauth2"
-  - "openid-connect"
```

Comment:
IMHO keeping `openid-connect` makes sense. We only this extension to help Keycloak users control Keycloak dynamically which is what they do with this admin client - but Keycloak is part of the OIDC setups. So if a KC user enters `openid-connect` this extension should be listed too

Reply:
@sberyozkin done

Reply:
@gsmet thanks